test/facility-loan/get-facility-loans-by-facility-identifier.api-test.ts

```diff
@@ -0,0 +1,127 @@
+import { PROPERTIES } from '@ukef/constants';
+import { DateStringTransformations } from '@ukef/modules/date/date-string.transformations';
+import { withAcbsAuthenticationApiTests } from '@ukef-test/common-tests/acbs-authentication-api-tests';
+import { IncorrectAuthArg, withClientAuthenticationTests } from '@ukef-test/common-tests/client-authentication-api-tests';
+import { Api } from '@ukef-test/support/api';
+import { ENVIRONMENT_VARIABLES, TIME_EXCEEDING_ACBS_TIMEOUT } from '@ukef-test/support/environment-variables';
+import { GetFacilityLoanGenerator } from '@ukef-test/support/generator/get-facility-loan-generator';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import nock from 'nock';
+
+describe('GET /facilities/{facilityIdentifier}/loans', () => {
+  const valueGenerator = new RandomValueGenerator();
+  const dateStringTransformations = new DateStringTransformations();
+  const portfolioIdentifier = PROPERTIES.GLOBAL.portfolioIdentifier;
+  const facilityIdentifier = valueGenerator.ukefId();
+  const getFacilityLoansUrl = `/api/v1/facilities/${facilityIdentifier}/loans`;
+
+  const { facilityLoansInAcbs, facilityLoansFromApi } = new GetFacilityLoanGenerator(valueGenerator, dateStringTransformations).generate({
+    numberToGenerate: 2,
```

Comment:
* 404
* Invalid facility ID
  *  ' '
  * ABC
  * !"£<

Reply:
- There's already a 404 test (see around line 73)
- It's a TODO/ticket (https://ukef-dtfs.atlassian.net/browse/APIM-202) to write reusable tests for this, as checking for invalid facility ID is common to several endpoints.